webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/esbuild.js

```diff
@@ -0,0 +1,45 @@
+// Load the go wasm.
+require("./go-wasm");
+
+class EsBuild extends Go {
+  constructor() {
+    super();
+    this.wasm = undefined;
+    this.workDir = "/app";
+    this.user = "esbuild";
+    this.homeDir = "/home/esbuild";
+    this.env.USER = this.user;
+    this.env.HOME = this.homeDir;
+    // process.cwd = () => this.workDir;
+  }
+
+  async load() {
+    const response = await fetch("esbuild.wasm");
```

Comment:
now that you are using `fetch` here, I have to bring in babel. `fetch` is definitely not supported in browser environments we're trying to support.

Reply:
fetch was browser first then made its way into node. It's 100% a browser supported method. https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch . It was meant to simplify XMLHTTPRequest.

Support since Chrome 42: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#browser_compatibility

Reply:
wasn't talking about `fetch` in particular but just in general our internal sources are growing so it's a good idea to use babel-loader for our sources only and then enforce a code style